Dockerfile

```diff
@@ -1,16 +1,17 @@
-FROM rocker/verse:3.6.1
+FROM rocker/verse:3.6.2
 MAINTAINER support@civisanalytics.com
 
 RUN DEBIAN_FRONTEND=noninteractive apt-get update -y --no-install-recommends && \
     apt-get install -y --no-install-recommends \
         curl \
         libudunits2-dev \
         libcairo-dev \
-        libgdal-dev \        
+        libgdal-dev \
         libgeos-dev \
         libglu1-mesa-dev \
         libx11-dev \
-        mesa-common-dev \        
+        mesa-common-dev \
+				libsodium-dev \
```

Comment:
de-indent and alphabetize, otherwise great!

Reply:
Fixed it! Dang tabs being inaccurate.